src/api/ready.js

```diff
@@ -0,0 +1,54 @@
+import data from '../util/data';
+import registry from '../global/registry';
+
+function ready (element) {
+  const components = registry.find(element);
+  for (let a = 0; a < components.length; a++) {
```

Comment:
I'd rather `const` it outside the loop, but yeah, may as well do that for older browsers :)